### PR TITLE
Techdebt/Move Login to Razor Page

### PR DESCRIPTION
**Pull request by acarbutt**
### Context
Move Login to Razor Page
### Changes proposed in this pull request
Move Login to Razor Page
### Checklist

- [x] Rebased master
- [x] Cleaned commit history
- [x] Tested by running locally

### CODE REVIEW

File: Frontend.Tests/PagesTests/Home/LoginTests.cs [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+namespace Frontend.Tests.PagesTests.Home
+{
+    public class LoginTests : PageTests
+    {
+        private readonly Login _subject;
+        private const string Username = "username";
+        private const string Password = "password";
+        private readonly Mock<IUrlHelper> _urlHelper = new Mock<IUrlHelper>();
+        public LoginTests()
```
Comment (StewartBarker) [inline Frontend.Tests/PagesTests/Home/LoginTests.cs:21]: Nice!

File: Frontend/Pages/Home/Login.cshtml [hunk 2]
```diff
@@ -1,12 +1,14 @@
+﻿@page "/home/login/{handler?}"
+@model Frontend.Pages.Home.Login
 @{
     ViewBag.Title = "Login";
     Layout = "_Layout";
 }
-
+<div asp-gds-validation-summary></div>
```
Comment (StewartBarker) [inline Frontend/Pages/Home/Login.cshtml:7]: Nice stuff switching to using validation tag helpers

File: Frontend/Pages/Home/Index.cshtml.cs [hunk 3]
```diff
@@ -3,6 +3,7 @@
 using Data;
 using Data.Models;
 using Frontend.ExtensionMethods;
+using Microsoft.AspNetCore.Authorization;
```
Comment (StewartBarker) [inline Frontend/Pages/Home/Index.cshtml.cs:6]: Prob not needed but we can just remove it another time

**Review by StewartBarker — APPROVED** [review]
Great stuff, tested locally and all looks good

File: Frontend/Pages/Home/Index.cshtml.cs (hunk 3 above)
Comment (acarbutt) [reply Frontend/Pages/Home/Index.cshtml.cs]: Thanks, removed

**Pull request merged**